index.html

```diff
@@ -124,14 +124,18 @@
         <p>See [[!RFC7230]] section 3.2.6 for definitions of `token` and `quoted-string`. This specification defines three NEL directives: <a>report-uri</a>, <a>max-age</a>, and <a>includeSubDomains</a>. The overall requirements for directives are:</p>
 
         <ul>
-          <li>The order of appearance of directives is not significant.</li>
-          <li>All directives MUST appear only once in an <a>NEL header field</a>. Directives are either optional or required, as stipulated in their definitions.</li>
+          <li>Directive may be optional or required.</li>
           <li>Directive names are case-insensitive.</li>
-          <li>User agents MUST ignore any <a>NEL header field</a> containing directives, or other header field value data, that does not conform to the syntax defined in this specification.</li>
-          <li>If an <a>NEL header field</a> contains directive(s) not recognized by the user agent, the user agent MUST ignore the unrecognized directives, and if the <a>NEL header field</a> otherwise satisfies the above requirements, the user agent MUST process the recognized directives.</li>
+          <li>Directive order is not significant.</li>
+          <li>Directive should be specified once.</li>
         </ul>
 
-        <p>Additional directives extending the semantic functionality of the <a>NEL header field</a> can be defined in other specifications, with a registry (having an IANA policy definition of IETF Review [[!RFC5226]]) defined for them at such time.</p>
+        <p>User agents MUST ignore any unknown or invalid directive(s), or other header field value data, that does not conform to the syntax defined in this specification. A valid <a>NEL header field</a> MUST, at a minimum, contain all of the "REQUIRED" directives defined in this specification.</p>

```

Comment:
FWIW, the text is the same, just in a different location -- see https://github.com/w3c/network-error-logging/pull/53/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL131.

"User agents MUST ignore any <a>NEL header field</a> containing directives, or other header field value data, that does not conform to the syntax defined in this specification."